packages/twenty-front/src/modules/spreadsheet-import/steps/components/MatchColumnsStep/MatchColumnsStep.tsx

```diff
@@ -261,13 +265,11 @@ export const MatchColumnsStep = <T extends string>({
       (column) => column.type === SpreadsheetColumnType.empty,
     );
     if (autoMapHeaders && isInitialColumnsState) {
-      const { matchedColumns } = getMatchedColumnsWithFuse(
```

Comment:
neat: this should be in a hook and this hook should be called synchronously when hitting "Import"